locale/en/docs/guides/backpressuring-in-streams.md

```diff
@@ -186,7 +186,7 @@ Let's take a look at a quick benchmark. Using the same example from above, we
 ran a few time trials to get a median time for both binaries.
 
 <!-- eslint-skip -->
```

Comment:
If we add linting for JS in Markdown, will this even be checked? If not I should also remove the `eslint` comments.

Reply:
To answer my question, they are not linted, so we can safely remove the eslint comments.